Bio/Seq.py

```diff
@@ -2887,8 +2934,6 @@ class _UndefinedSequenceData(SequenceDataAbstractBaseClass):
 
     def __init__(self, length):
         """Initialize the object with the sequence length."""
-        if length < 0:
-            raise ValueError("Length must not be negative.")
```

Comment:
Why remove this test?

Reply:
An `_UndefinedSequenceData` is created only for positive values of `length`.

Reply:
If this was a public class, I would want the check the length was positive. But for a private class, perhaps it is not needed - but at least document this requirement in the docstring?